### PR TITLE
[HOT-43] CreatePotViewSwticher component

### PR DESCRIPTION
**Pull request by Keerush**
JIRA: https://hotpotbuddy.atlassian.net/browse/HOT-43 
Created component for pot view switching, and added it to Pot Instance page.

image

### CODE REVIEW

File: client/src/pages/pot-instance/PotInstance.tsx [hunk 1]
```diff
@@ -1,9 +1,11 @@
...
 import { Button } from '@mui/material';
 import LandingPage from 'pages/pot-instance/components/LandingPage/LandingPage';
-import PotView, {
-  PotViewState
-} from 'pages/pot-instance/components/PotView/PotView';
+import {
+  PotView,
+  PotViewState,
+  PotViewSwitcher
+} from 'pages/pot-instance/components/PotView';
```
Comment (AndrewLin95) [inline client/src/pages/pot-instance/PotInstance.tsx:8]: oo nice exporting

File: client/src/pages/pot-instance/components/PotView/PotViewSwitcher.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+interface Props {
+  selectedValue: PotViewState;
+  onChange: (value: keyof typeof PotViewState) => void;
+}
+
+const PotViewSwitcher = ({ selectedValue, onChange }: Props) => {
+  const handleChange = (
+    event: React.MouseEvent<HTMLElement>,
+    newValue: keyof typeof PotViewState
```
Comment (AndrewLin95) [inline client/src/pages/pot-instance/components/PotView/PotViewSwitcher.tsx:14]: keyof typeof?

**Review by AndrewLin95 — APPROVED** [review]
Noice

File: client/src/pages/pot-instance/components/PotView/PotViewSwitcher.tsx (hunk 2 above)
Comment (Keerush) [reply client/src/pages/pot-instance/components/PotView/PotViewSwitcher.tsx]: Forgot to remove those, playing around with enum types.

**Pull request merged**